lib/cypress-rails/finds_bin.rb

```diff
@@ -4,8 +4,8 @@ module CypressRails
   class FindsBin
     LOCAL_PATH = "node_modules/.bin/cypress"
 
-    def call(dir = Dir.pwd)
-      local_path = Pathname.new(dir).join(LOCAL_PATH)
+    def call(cy_dir = Dir.pwd)
```

Comment:
Not entirely sure these variable name changes are necessary, but I just wanted it to be more clear that we're looking for a bin path within  a directory with Cypress.